src/js/components/DataTable/DataTable.js

```diff
@@ -206,6 +208,19 @@ const DataTable = ({
     if (nextScrollOffset !== scrollOffset) setScrollOffset(nextScrollOffset);
   });
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useLayoutEffect(() => {
+    if (placeholder && placeholderRef.current) {
+      const nextHeaderHeight = headerRef.current
+        ? headerRef.current.getBoundingClientRect().height : 0;
+      const nextFooterHeight = footerRef.current
+        ? footerRef.current.getBoundingClientRect().height : 0;
+      const nextMinHeight = nextHeaderHeight + nextFooterHeight +
+        placeholderRef.current.getBoundingClientRect().height;
+      if (minHeight !== nextMinHeight) setMinHeight(nextMinHeight);
```

Comment:
Rather than cause a re-render, I'm wondering if we should just set the `min-height` directly on the `placeholderRef.current` here.

Reply:
I'll test that idea